jsonschema-core.xml

```diff
@@ -2095,6 +2101,440 @@
+]]>
+                        </artwork>
+                    </figure>
+                    <t>
+                        The JSON key for this information is "keywordLocation".
+                    </t>
+                </section>
+
+                <section title="Keyword Absolute Location">
+                    <t>
+                        The absolute, dereferenced location of the validating keyword.  The value MUST
+                        be expressed as an absolute URI, and it MUST NOT include "$ref" or
+                        "$recursiveRef" segments.
+                    </t>
+                    <figure>
+                        <artwork>
+<![CDATA[
+http://json-schema.org/draft-08/schema#/$defs/nonNegativeInteger/minimum
+]]>
+                        </artwork>
+                    </figure>
+                    <t>
+                        This information MAY be omitted if either the relative location contains no
+                        references or if the schema does not declare an absolute URI as its "$id".
+                    </t>
+                    <t>
+                        The JSON key for this information is "absoluteKeywordLocation".
+                    </t>
+                </section>
+
+                <section title="Instance Location">
+                    <t>
+                        The location of the JSON value within the instance being validated.  The
+                        value MUST be expressed as a JSON Pointer.
+                    </t>
+                    <t>
+                        The JSON key for this information is "instanceLocation".
+                    </t>
+                </section>
+
+                <section title="Error or Annotation">
+                    <t>
+                        The error or annotation that is produced by the validation.
+                    </t>
+                    <t>
+                        For errors, the specific wording for the message is not defined by this
+                        specification.  Implementations will need to provide this.
+                    </t>
+                    <t>
+                        The JSON key for failed validations is "error"; for successful validations
+                        it is "annotation".
+                    </t>
+                </section>
+
+                <section title="Nested Results">
+                    <t>
+                        For the two hierarchical structures, this property will hold nested errors
+                        and annotations.
+                    </t>
+                    <t>
+                        The JSON key for nested results in failed validations is "errors"; for
+                        successful validations it is "annotations".
+                    </t>
+                </section>
+
+            </section>
+
+            <section title="Output Structure">
+                <t>
+                    The output MUST be an object containing two properties
```

Comment:
I meant also as in along with the Pass/Fail section below. I felt the use of MUST here with "containing two properties" may confuse people when really the only required property is `valid`. Reading this section it feels like it could cause confusion. Given the description up in "Output Levels" says to look at this section for details of the output structure it felt like more information was needed here as well as in the Pass/Fail section or at least a reference to it.

Reply:
✔️